### DISCUSSION TITLE
[Script request] updated version of qBittorrent-nox

### DISCUSSION THREAD

**Discussion by damnsanga** [community-scripts/ProxmoxVE#568] [Request script]
### Application Name

qBittorrent-Nox

### Website

https://github.com/userdocs/qbittorrent-nox-static

### Description

Is it possible to update the tteck script to use qbittorent-nox version 5.X based on github project qbittorrent-nox-static ?
This project is more active with updates to qbittorrent-nox.

The qbittorrent-nox-static project is a bash build script that compiles a static qbittorrent-nox binary using the latest available dependencies from their source. These statically linked binaries can run on any matching CPU architecture and are not OS specific. This means you can run a x86_64 Alpine edge build on any Linux based OS of like CentOS | Fedora | OpenSuse | Debian | Ubuntu and more.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by DeclanE47** [comment]
This would be great to have updated to [5.0.3]

**Comment by Aloe-recite** [comment]
adding my vote! 🙏🏻

**Comment by WinterWolf-ahc** [comment]
And my axe!

**Reply by DeclanE47** [reply]
That got a giggle out of me 🥇

**Comment by kafisatz** [comment]
below is my 'script' to achieve this.
Please note that the LXC (by default) puts the executable in `/usr/bin/qbittorrent-nox`
```
systemctl stop qbittorrent-nox.service

wget -qO /usr/bin/qbittorrent-nox https://github.com/userdocs/qbittorrent-nox-static/releases/latest/download/x86_64-qbittorrent-nox

systemctl start qbittorrent-nox.service


```

**Reply by DeclanE47** [reply]
@kafisatz That is working perfectly, Thank you for that :)

**Reply by Gangstrocity** [reply]
Thanks so much for this. Worked perfectly and it was quick too.

**Comment by SirJohnDoe** [comment]
Alternatively an Alpine Linux version of the script would also be welcome, as it's lightweight and it would make updating easier (with the cron LXC update script). The Alpine Linux version seems to be up-to-date unlike the latest bookworm release: https://pkgs.alpinelinux.org/package/edge/community/x86/qbittorrent-nox

For now I've installed the qbittorrent-nox-static one thanks to kafisatz :)

**Comment by kuzyn** [comment]
See https://github.com/community-scripts/ProxmoxVE/issues/15876#issuecomment-5011953781 for an explanation. The v4 to v5  guard is in place to avoid compatibility and performance issues.
